SecurityPkg/Tcg/Tcg2Acpi/Tcg2Acpi.c

```diff
@@ -687,14 +687,21 @@ PublishAcpiTable (
   // Otherwise, the PCR record would be different after TPM FW update
   // or the PCD configuration change.
   //
-  TpmMeasureAndLogData (
-    0,
-    EV_POST_CODE,
-    EV_POSTCODE_INFO_ACPI_DATA,
-    ACPI_DATA_LEN,
-    Table,
-    TableSize
-    );
+  // MU_CHANGE [BEGIN]
+  // Allow a platform to drop TCG ACPI measurements until we have a chance to make them more
+  // consistent and functional.

```

Comment:
Is this still needed? Did we ever make these measurements more consistent and functional? @spbrogan @makubacki @apop5?

Reply:
As discussed offline, this is required for certain scenarios and needs to be kept for the time being.